src/main/java/nl/knaw/dans/ingest/core/ImportArea.java

```diff
@@ -83,6 +89,14 @@ public String startImport(Path inputPath, boolean isBatch, boolean continuePrevi
         return relativeInputDir.toString();
     }
 
+    private Boolean isSafeInputPath(Path inputPath) {
```

Comment:
* Why `Boolean` instead of `boolean`?
* `if (<somecondition>) return true else false` can be simplified to `<somecondition>`

More importantly still, you should replace this function with one that throws the `IllegalArgumentException` itself, instead of returning a boolean. You could call it  `checkDirectoryInInbox` so it properly describes what it checks.